core/domain/improvements_domain_test.py

```diff
@@ -72,6 +72,7 @@ def test_to_dict_has_expected_value(self) -> None:
             feconf.DEFAULT_INIT_STATE_NAME, 'issue description',
             constants.TASK_STATUS_RESOLVED, self.owner_id,
             self.MOCK_DATE)
+        self.maxDiff = None
```

Comment:
Is this needed or did you use it just for testing?

Reply:
Removed